sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DataSourceV2ScanExecBase.scala

```diff
@@ -43,7 +44,32 @@ trait DataSourceV2ScanExecBase extends LeafExecNode {
   override def simpleString(maxFields: Int): String = {
     val result =
       s"$nodeName${truncatedString(output, "[", ", ", "]", maxFields)} ${scan.description()}"
-    Utils.redact(sqlContext.sessionState.conf.stringRedactionPattern, result)
+    redact(result)
+  }
+
+  /**
+   * Redact the sensitive information in the given string.
```

Comment:
Shall we use the consistent description since this is a copy of `DataSourceScanExec.redact`?
```
  /**
   * Shorthand for calling redactString() without specifying redacting rules
   */
```

Reply:
@dongjoon-hyun Sure.